DataFormats/PatCandidates/src/PackedCandidate.cc

```diff
@@ -92,6 +68,66 @@ void pat::PackedCandidate::unpack() const {
       p4c.release();
     }
 }
+
+/*void pat::PackedCandidate::legacyCovarianceUnpacking() const {
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant